src/Builder/FormContractor.php

```diff
@@ -196,13 +193,7 @@ private function hasAssociation(FieldDescriptionInterface $fieldDescription)
         ], true);
     }
 
-    /**
-     * @param string $type
-     * @param array  $classes
-     *
-     * @return array
-     */
-    private function checkFormClass($type, $classes)
+    private function checkFormClass(string $type, array $classes): array
     {
         return array_filter($classes, static function ($subclass) use ($type) {
```

Comment:
```suggestion
        return array_filter($classes, static function (string $subclass) use ($type): bool {
```

Reply:
The `checkFormClass` return value is only used as truthy/falsy.
So I changed the method.

WDYT ? Should I rename the function too ? `isFormClass` ?